ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/default/provision/cucushift-installer-rehearse-ibmcloud-ipi-default-provision-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-default-provision
+  steps:
+  - ref: ibmcloud-provision-iamservice-minimal-permission
```

Comment:
If you do not have plan to enable minimal permission on all install workflow, you an drop this step, or update the install workflow name with `mini-perm` keyword.